sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 #### Other Changes
 * Added temporary internal-only option to enable thin client mode with system property COSMOS.THINCLIENT_ENABLED, setting the thin client endpoint with system property COSMOS.THINCLIENT_ENDPOINT, and default thin client endpoint with system property COSMOS.DEFAULT_THINCLIENT_ENDPOINT while the thin-client transport is still under development. This transport mode is not yet supported or ready to be used by external customers. Please don't use these configs in any production scenario yet. - [PR 43188](https://github.com/Azure/azure-sdk-for-java/pull/43188)
+* Added the `vectorIndexShardKey` to the vectorIndexSpec for DiskANN vector search. - [PR 44007](https://github.com/Azure/azure-sdk-for-java/pull/44007)
```

Comment:
```suggestion
* Added the `vectorIndexShardKey` to the vectorIndexSpec for QuantizedFlat and DiskANN vector search. - [PR 44007](https://github.com/Azure/azure-sdk-for-java/pull/44007)
```

Reply:
Done

Reply:
I don't see hte change here?  Also, update to pluralize the key. 

```suggestion
* Added the `vectorIndexShardKeys` to the vectorIndexSpec for Quantized Flat and DiskANN vector search. - [PR 44007](https://github.com/Azure/azure-sdk-for-java/pull/44007)
```